includes/class-edd-cli.php

```diff
@@ -1039,7 +1039,7 @@ public function migrate_order_notes( $args, $assoc_args ) {
 
 			$progress->finish();
 
-			WP_CLI::line( __( 'Migration complete.', 'easy-digital-downloads' ) );
+			WP_CLI::line( __( 'Migration complete: Notes', 'easy-digital-downloads' ) );
```

Comment:
How about changing this to `Order Notes` ? It would need to be updated in a few other places throughout this method too.

Just thinking we also have `Customer Notes` so it makes sense to prefix this one with `Order` too.